src/server/req_signal.c

```diff
@@ -508,6 +508,8 @@ create_resreleased(job *pjob)
 	char *resreleased;
 	attribute reqrel;
 	char buf[1024] = {0};
+	char *dflt_ncpus_rel = ":ncpus=0";
+	int no_res_rel = 1;
```

Comment:
It doesn't hurt, but there is no need to initialize this variable.  It'll pick up it's value when you enter the loop.